hub.py

```diff
@@ -240,8 +240,37 @@ def get_generated_config(self, auth_provider, proxy_secret_key):
             # these can all exist fine.
             generated_config['jupyterhub']['hub']['config']['GenericOAuthenticator'] = auth_provider.get_client_creds(client, self.spec['auth0']['connection'])
 
+            # Add 2i2c staff to the list of admins and allowed users
+            admin_users, allowed_users = self.get_hub_allowed_users()
+            generated_config['jupyterhub']['hub']['config']['Authenticator'] = {
+                'admin_users': admin_users,
+                'allowed_users': allowed_users
+            }
+
         return self.apply_hub_template_fixes(generated_config, proxy_secret_key)
 
+    def get_hub_allowed_users(self):
```

Comment:
Can you put this in `hubs.yaml`? `clusters` is a top level variable, this could be another top level.

Reply:
The `staff_emails` and `staff_github_ids`, rigth?